pkg/natdiscovery/listener.go

```diff
@@ -0,0 +1,73 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package natdiscovery
+
+import (
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/proto"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+
+	natproto "github.com/submariner-io/submariner/pkg/natdiscovery/proto"
+)
+
+func (nd *natDiscovery) runListener(stopCh <-chan struct{}) error {
+	serverAddress, err := net.ResolveUDPAddr("udp", ":"+strconv.Itoa(natproto.DefaultPort))
+	if err != nil {
+		return errors.Wrap(err, "Error resolving UDP address")
```

Comment:
The linter didn't catch this.

```suggestion
		return errors.Wrap(err, "error resolving UDP address")
```

Reply:
Done